state/migration_export.go

```diff
@@ -122,6 +123,11 @@ func (st *State) exportImpl(cfg ExportConfig) (description.Model, error) {
 	if !found && !cfg.SkipSettings {
 		return nil, errors.New("missing model config")
 	}
+	// Ensure mode is set so importing model doesn't
+	// default to a value not valid in 2.9
+	if _, ok := modelConfig.Settings[config.ModeKey]; !ok {
```

Comment:
Hmmm, seeing this change in practice as opposed to discussing it seems ick. Need to think a bit....